### ISSUE TITLE
Improved Documentation: Currently documentation is inaccurate and very messy.

### ISSUE THREAD

**Issue by c0fe** [frappe/erpnext#35511]
I am going to open a new ticket since the other one was closed as not something to be fixed and incorrectly stated that a required component was listed in the documentation, it wasn't unless you read the instructions on how to install the programs individually.

This brings me to the purpose of this issue. The documentation is a mess, it is incomplete and pertains errors. ERPNext includes two variants, Docker and Manual, the manual has one step, install Frappe. This is the equivalent of saying that the only step to installing a .Net program is to install .Net framework. You need to expand on that, saying that being the only documented step is obviously wrong.

Frappe installation is messy as well, you have to go through 3 links for some reason to get to a website that includes the instructions. Docker installation is a mess, not to mention using markdown files on Github instead of an already made and significantly better designed documentation site is very strange. Overall the whole thing is just all over the place and skips several steps.

I would offer to rewrite but the reception to me bringing this issue up in the first place has been very poor. So the suggestion is this, fix your documentation, it is a massive mess rather than deny it is.

**Comment by abnairs** [comment]
I agree with @c0fe . Frappe documentation need lot of improvement. Documentation targeting developers is required.
I suggest following documentation: 

1.  A tutorial similar to this django mdn tutorial - https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Tutorial_local_library_website. Right from installation to deployment. Should include both the backend and frontend part.
2. Documentation on using JINJA with frappe. Something like this - https://docs.djangoproject.com/en/5.1/#the-template-layer
3.  A comprehensive documentation on what are portal pages, their need, how to use them. Something like this - https://docs.djangoproject.com/en/5.1/#the-view-layer
4. A comprehensive documentation on url routes. For example - https://docs.djangoproject.com/en/5.1/topics/http/urls/
5. A comprehensive documentation on ORM. For example - https://docs.djangoproject.com/en/5.1/topics/db/queries/

I think these are bare minimum developer documentation that is required for any developer starting out with frappe. Plus focus should be creating text based tutorial instead of video based tutorial. Video based tutorial are time consuming, one cannot jump directly to the required part, searching for topic within a video tutorial is non existent.

Most of the tutorials or guides in frappe school is targeting erpnext. 

And I am ready to contribute as well.
